frontend/app/src/components/inputs/AmountInput.vue

```diff
@@ -1,6 +1,7 @@
 <script setup lang="ts">
 import { useListeners } from 'vue';
 import IMask, { type InputMask } from 'imask';
+import { VTextField } from 'vuetify/lib/components';
```

Comment:
Any way we can avoid importing from `vuetify/lib`?

This import will be problematic during migration to vuetify 3